pkg/local_workflows/ignore_workflow/config.go

```diff
@@ -70,6 +100,26 @@ func getOrgIgnoreApprovalEnabled(engine workflow.Engine) configuration.DefaultVa
 	}
 }
 
+func getOrgIgnoreReasonRequired(engine workflow.Engine) configuration.DefaultValueFunction {
```

Comment:
suggest: `getOrgIgnoreReasonRequired`, `getOrgIgnoreSettingsConfig`, `getOrgIgnoreApprovalEnabled` are all making use of `getOrgIgnoreSettings`. Is `getOrgIgnoreSettings` something that can be added to config via `config.AddDefaultValue`? That way we can take advangate of caching to save mulutiple API calls.